### PR TITLE
sw-6171 edit bond test #minor

### PR DESCRIPTION
**Pull request by Nicholassully**

### CODE REVIEW

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withBond();
+});
+
+describe(
+  "Edit bond",
+  () => {
+    Cypress._.times(50, () => {
```
Comment (kate-49) [inline cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:11]: probably want to take the 50 times run out before the merge

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        cy.get("@client").then(({id: clientId}) => {
+          cy.visit(
+            `/supervision/#/clients/${clientId}`
+          );
+        });
+
+        cy.get(".TABS_ORDERS").click();
+        cy.get("#tab-order-list-in-page-notification").should("contain", "Please wait...");
+        cy.get("#tab-order-list-in-page-notification").should("not.contain", "Please wait...");
+
```
Comment (kate-49) [inline cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:22]: Is this just to wait for it to load? It's a little confusing we want it to contain but also not contain the words

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        cy.get('#requiredBondAmount').clear();
+        cy.get('#requiredBondAmount').type("999999999999")
+        cy.contains("button", "Save & exit").click();
+        cy.get(".validation-summary")
+          .and("contain.text", "There is a problem")
+          .and("contain.text", "Required bond amount - The input is not less than '10000000000'");
+      });
+    });
+  }
+);
```
Comment (kate-49) [inline cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:33]: It doesn't look like there's a test for the original 'the bond should look as expected', is this shown anywhere else?

Comment (Nicholassully) [reply cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:33]: Yeah I thought it was this file Dispense bond

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js (hunk 2 above)
Comment (Nicholassully) [reply cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:22]: Aye it is indeed

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js (hunk 1 above)
Comment (Nicholassully) [reply cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:11]: oops nice catch thank you

File: cypress/e2e/supervision/orders/bonds/edit-bond.cy.js (hunk 3 above)
Comment (kate-49) [reply cypress/e2e/supervision/orders/bonds/edit-bond.cy.js:31]: So I think dispense bond shows that the bond loads and you can dispense it but the original Sirius test checks that you can edit it and then that info gets displayed. Is it possible to test that too?

**Review by kate-49 — APPROVED** [review]

**Pull request merged**